### PR TITLE
test: fixed sdk/multiple_installations/cjs/test.js for Node v23

### PR DESCRIPTION
**Pull request by kirrg001**

### CODE REVIEW

File: packages/collector/test/tracing/sdk/multiple_installations/cjs/test.js [hunk 1]
```diff
@@ -69,43 +92,44 @@ mochaSuiteFn('[CJS] tracing/sdk/multiple_installations', function () {
...
-      withError: false,
-      pid: String(controls.getPid()),
-      extraTests: span => {
-        expect(span.data.sdk.name).to.eql('intermediate-span-name');
-      }
+    return testUtils.retry(async () => {
+      spans = await agentControls.getSpans();
+
+      // 2x SDK, 1x HTTP ENTRY
+      expect(spans.length).to.eql(3);
```
Comment (kirrg001) [inline packages/collector/test/tracing/sdk/multiple_installations/cjs/test.js:100]: FYI We received 2 spans only on prerelease pipeline.

File: packages/collector/test/tracing/sdk/multiple_installations/cjs/test.js [hunk 2]
```diff
@@ -33,7 +33,30 @@ mochaSuiteFn('[CJS] tracing/sdk/multiple_installations', function () {
   let controls;
 
   before(async () => {
-    testUtils.runCommandSync('npm install --production --no-optional --no-audit @instana/collector', tmpDir);
+    /**
+     * NOTE: We install the current code base, not the codebase from npm.
+     *       Because otherwise there could be a mismatch between the logic
+     *       e.g. We added Node v23 support on main, we pull the latest version from npm
+     *       and tracing won't work.
```
[the remark this answers is not in the document]
Comment (abhilash-sivan) [reply packages/collector/test/tracing/sdk/multiple_installations/cjs/test.js:40]: we may need this in packages/collector/test/tracing/sdk/multiple_installations/esm/test.js in as well.. I will look into it. LGTM

**Review by abhilash-sivan — APPROVED** [review]

File: packages/collector/test/tracing/sdk/multiple_installations/cjs/test.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (kirrg001) [reply packages/collector/test/tracing/sdk/multiple_installations/cjs/test.js:40]: Thanks!

[the remark this answers is not in the document]
Comment (aryamohanan) [reply packages/collector/test/tracing/sdk/multiple_installations/cjs/test.js:40]: make sense! Thanks for the comments ❤️

**Review by aryamohanan — APPROVED** [review]
Added a minor comment. LGTM

File: packages/collector/test/tracing/sdk/multiple_installations/cjs/src/app.js [hunk 3]
```diff
@@ -55,7 +54,10 @@ async function createSDKSpans() {
 }
 
 app.get('/', (req, res) => res.status(200).send('OK'));
-app.get('/trace', (req, res) => res.status(200).send('OK'));
+app.get('/trace', (req, res) => {
+  log('/trace');
```
[the remark this answers is not in the document]
Comment (kirrg001) [reply packages/collector/test/tracing/sdk/multiple_installations/cjs/src/app.js:58]: Yes. It helps when working on the test.

**Pull request merged**